src/plugins/StatusBar/StatusBar.js

```diff
@@ -119,11 +123,11 @@ module.exports = (props) => {
   const isHidden = (uploadState === STATE_WAITING && props.hideUploadButton) ||
     (uploadState === STATE_WAITING && !props.newFiles > 0)
 
```

Comment:
Could we re-use `isHidden` constant we already have in place here, instead of adding a new `hiddenClass ` css class? Perhaps by adding one extra condition, like so (haven’t tested this):

```js
const isHidden = (uploadState === STATE_WAITING && props.hideUploadButton) ||
  (uploadState === STATE_WAITING && !props.newFiles > 0) ||
  (uploadState === STATE_COMPLETE && props.hideAfterFinish)
```